cdap-data-fabric/src/main/java/io/cdap/cdap/data2/dataset2/lib/cube/DefaultCube.java

```diff
@@ -439,30 +442,47 @@ private void incrementMetric(String metricName, long value) {
     return result;
   }
 
-  private Collection<TimeSeries> convertToQueryResult(CubeQuery query,
-                                                      Table<Map<String, String>, String,
-                                                        Map<Long, Long>> resultTable) {
-
-    List<TimeSeries> result = Lists.newArrayList();
+  private Collection<TimeSeries> convertToQueryResult(
+    CubeQuery query, Table<Map<String, String>, String, Map<Long, Long>> resultTable) {
```

Comment:
this indentation looks odd. why change it?

Reply:
The original indentation is indented on the table type, which I think is confusing to read, so I put the variable in the same line.